### PR TITLE
[ALZ-167] Surface previous submission if available

### PR DESCRIPTION
**Pull request by erichare**
Based on the submission id and the user, set the 3 fields in the review page plus the text of the submit button accordingly

### CODE REVIEW

File: R/mod-panel-section.R [hunk 1]
```diff
@@ -148,6 +148,30 @@ mod_panel_section_server <- function(input, output, session, synapse, syn, user,
...
+      result <- readr::read_csv(
+        syn$tableQuery(
+          glue::glue(
+            query
+          )
+        )$filepath
+      )
+
+      if (nrow(result) > 0) {
+        updateTextInput(session, "overall_score", value = result$overall_score[1])
```
Comment (thomasyu888) [inline R/mod-panel-section.R:165]: I'm coming at this from a naive angle.  Do we ever expect more than one to be returned?

**Review by thomasyu888 — APPROVED** [review]
LGTM - just one comment, but I'll let Jess be the final reviewer

**Comment by erichare** [comment]
> LGTM - just one comment, but I'll let Jess be the final reviewer

Thank's @thomasyu888 ! that's a great point actually - maybe the logic should be always use the most recent one if there are multiple? But yes, i'll let @JessterB comment - we're syncing up this afternoon as well.

Thanks for the review!

File: R/mod-panel-section.R (hunk 1 above)
Comment (JessterB) [reply R/mod-panel-section.R:165]: @thomasyu888 No, there is only one overall score per submission (form_data_id).

@erichare We shouldn't filter on `scorer` for queries to the Submissions table; it doesn't matter who added/updated the singular row for a given submission, if it exists we want it.

**Review by JessterB — CHANGES_REQUESTED** [review]

**Comment by erichare** [comment]
@JessterB updated this! my mistake.

**Review by JessterB — APPROVED** [review]

**Pull request merged**